.github/actions/bootstrap/action.yaml

```diff
@@ -24,13 +25,13 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - uses: actions/setup-go@v4
+    - uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe #v4.1.0
       with:
         go-version: ${{ inputs.go-version }}
 
     - name: Restore tool cache
       id: tool-cache
-      uses: actions/cache@v3
+      uses: actions/cache@704facf57e6136b1bc63b828d79edcd491f0ee84 #v3.3.2
```

Comment:
It would be nice if there was a mechanism besides the comment; does any automated process ensure that the version in the comment and the version identified by the digest are the same?

Reply:
^ Dependabot will actually update the commit and the commented version - let me find the PR that shows that example

Reply:
Grype example where the sha and version comment are updated:
https://github.com/anchore/grype/pull/1481/files